mongodb/scripts/db_seed.py

```diff
@@ -0,0 +1,112 @@
+import pandas as pd  
+from pymongo import MongoClient 
+from dotenv import load_dotenv
+import os 
+
+def check_if_not_empty(coll,coll_name):  
+
+    if coll.count_documents({}) > 0: 
+
+        print("INFO: Data found in %s collection ... skipping inserting data in this collection" %(coll_name)) 
+        return True  
+    
+    print("INFO: No data found in %s collection ... inserting data" %(coll_name))
+    return False
+
+def load_users(coll,admin_1,admin_2): 
```

Comment:
Introducing the new isAdmin field broke the users endpoints, I think this may be resolved with the other PR that is up but hard to say until both of them get merged to dev

Request: http://localhost:8000/users
Response:
![image](https://github.com/SENG-499-Company2-B01/Backend/assets/69998437/f0f0252f-595f-4bbd-8e2c-61eacc69830f)

Everything else worked good though!

Reply:
I don't think it's the isAdmin field it's complaning about... looks like my qualifications are being read in as a string from the csv file... not as an array of strings. I can quickly fix that

Reply:
![image](https://github.com/SENG-499-Company2-B01/Backend/assets/44540913/040e0dc9-3f9f-48a2-96af-2d26d2a23bc0)

Reply:
Nice! and yeah reading is hard, didn't even realize the error it gave 😆